ds3/ds3network.py

```diff
@@ -207,7 +207,7 @@ def send_request(self, request):
             if key == 'Content-Length':
                 # Add to headers,  but not to amz-headers
                 headers[key] = value
-            elif key in ['Content-CRC32', 'Content-CRC32C', 'Content-MD5', 'Content-SHA256', 'Content-SHA512']:
+            elif key in ['Content-CRC32', 'Content-CRC32C', 'Content-MD5', 'Content-SHA256', 'Content-SHA512','Range','Content-Length','Accept-Ranges','Content-Range']:
```

Comment:
The if above covers the `Content-Length`. Also, why add `Accept-Ranges` and `Content-Range`?